crates/fluvio-sc/src/k8/controllers/managed_connector_deployment.rs

```diff
@@ -206,20 +236,43 @@ impl ManagedConnectorDeploymentController {
             "--".to_string(),
             format!("--fluvio-topic={}", mc_spec.topic),
         ];
+        let type_ = &mc_spec.type_;
+        let image = if type_.starts_with("https://") {
```

Comment:
need option to disable third-party connector

Reply:
The option is that there are no `--connector-prefix`s specified.

Reply:
where is used?

Reply:
Oh, it's used at the top level cluster start CLI. If you don't specify any `--connector-prefix`, the third party connectors are never used.

Reply:
got it. make sense